src/components/hero/_macro-options.md

```diff
@@ -9,10 +9,10 @@
 | html                    | string                                                    | false    | Allows arbitrary HTML for additional content to be added to the component                                                          |
 | detailsColumns          | integer                                                   | false    | Number of grid columns for the hero to span on screens larger than the medium breakpoint, defaults to 8                            |
 | descriptionList         | `DescriptionList` [_(ref)_](/components/description-list) | false    | Settings to set the DescriptionList component within the HTML `<hero>` element                                                     |
-| officialStatisticsBadge | boolean                                                   | false    | Set to “true” display the official statistics badge in the hero                                                                    |
+| officialStatisticsBadge | boolean                                                   | false    | Set to “true” display the official statistics badge in the "grey" hero variant                                                     |
```

Comment:
I would word this differently to be more specific. Maybe something like this:

Set to “true” display the official statistics badge (only available for the "grey" hero variant)

Same with the census logo